### PR TITLE
Update decomposition methods for SymPy 1.1

### LINKED ISSUE #812: matrix factorizations may change names in dev sympy
Old names are deprecated so we'll probably get lots of warnings.

### PR DESCRIPTION
**Pull request by mtmiller**
Prefer the new SymPy method names 'cholesky_decomposition',
'LU_decomposition', and 'QR_decomposition' to the now-deprecated names.

Fixes #812.

Tested locally against Python 3.6.1 with SymPy 1.1 dev and against Python 3.5.3 with SymPy 1.0.

### CODE REVIEW

File: inst/@sym/chol.m [hunk 1]
```diff
@@ -66,7 +66,15 @@
   elseif (nargin > 2)
     print_usage ();
   end
-  y = python_cmd('return _ins[0].cholesky(),', x);
+
+  cmd = { 'A = _ins[0]' ...
+          'try:' ...
+          '    y = A.cholesky_decomposition()' ...
```
Comment (mtmiller) [inline inst/@sym/chol.m]: This new variable could be named `L` if you like

File: inst/@sym/chol.m [hunk 2]
```diff
@@ -66,7 +66,15 @@
...
+          'except AttributeError:' ...
+          '    y = A.cholesky()' ...
+          'return y' };
```
Comment (mtmiller) [inline inst/@sym/chol.m]: I see `return foo,` in some functions that return a single value, is there a preference for with or without the comma?

Comment (cbm755) [inline inst/@sym/chol.m]: Preference for without.

(Comma used to be required, someone convinced me to drop it, maybe @latot).

File: inst/@sym/chol.m (hunk 1 above)
Comment (cbm755) [inline inst/@sym/chol.m]: I probably would, but I'm not bothered either way

**Comment by cbm755** [comment]
LGTM, ok to merge?  (its marked WIP)/

**Comment by mtmiller** [comment]
Ok

**Comment by cbm755** [comment]
Some churn upstream, will wait a bit and see what happens before merging.

Thanks for this.

**Comment by mtmiller** [comment]
The upstream name changes were reverted for SymPy 1.1, no longer needed.

**Pull request closed without merging**